#10x downsample
fgbio ClipBam --input=MJ09.ManJav2.0.ds10x.bam --output=MJ09.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=MJ09.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=MJ16.ManJav2.0.ds10x.bam --output=MJ16.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=MJ16.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=MJ17.ManJav2.0.ds10x.bam --output=MJ17.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=MJ17.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=MJ26.ManJav2.0.ds10x.bam --output=MJ26.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=MJ26.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=MJav77.ManJav2.0.ds10x.bam --output=MJav77.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=MJav77.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=MP11.ManJav2.0.ds10x.bam --output=MP11.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=MP11.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=MPent76.ManJav2.0.ds10x.bam --output=MPent76.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=MPent76.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10594.ManJav2.0.ds10x.bam --output=P10594.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10594.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10595.ManJav2.0.ds10x.bam --output=P10595.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10595.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10596.ManJav2.0.ds10x.bam --output=P10596.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10596.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10597.ManJav2.0.ds10x.bam --output=P10597.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10597.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10598.ManJav2.0.ds10x.bam --output=P10598.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10598.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10599.ManJav2.0.ds10x.bam --output=P10599.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10599.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10600.ManJav2.0.ds10x.bam --output=P10600.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10600.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10601.ManJav2.0.ds10x.bam --output=P10601.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10601.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10602.ManJav2.0.ds10x.bam --output=P10602.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10602.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10603.ManJav2.0.ds10x.bam --output=P10603.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10603.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=TRI.ManJav2.0.ds10x.bam --output=TRI.ManJav2.0.ds10x.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=TRI.ManJav2.0.ds10x_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true


#15x downsample
fgbio ClipBam --input=P10598.ManJav2.0.bam --output=P10598.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10598.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10600.ManJav2.0.bam --output=P10600.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10600.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10596.ManJav2.0.ds.bam --output=P10596.ManJav2.0.ds.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10596.ManJav2.0.ds_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10597.ManJav2.0.ds.bam --output=P10597.ManJav2.0.ds.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10597.ManJav2.0.ds_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10599.ManJav2.0.ds.bam --output=P10599.ManJav2.0.ds.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10599.ManJav2.0.ds_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10601.ManJav2.0.ds.bam --output=P10601.ManJav2.0.ds.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10601.ManJav2.0.ds_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10602.ManJav2.0.ds.bam --output=P10602.ManJav2.0.ds.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10602.ManJav2.0.ds_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P10603.ManJav2.0.ds.bam --output=P10603.ManJav2.0.ds.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P10603.ManJav2.0.ds_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true

fgbio ClipBam --input=TRI.ManJav2.0.ds.bam --output=TRI.ManJav2.0.ds.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=tri.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true

fgbio ClipBam --input=MP11.ManJav2.0.bam --output=MP11.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=MP11.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=MJ09.ManJav2.0.bam --output=MJ09.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=MJ09.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=MJ16.ManJav2.0.bam --output=MJ16.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=MJ16.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=MJ17.ManJav2.0.bam --output=MJ17.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=MJ17.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=MJ26.ManJav2.0.bam --output=MJ26.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=MJ26.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true

fgbio ClipBam --input=MPent76.ManJav2.0.ds.bam --output=MPent76.ManJav2.0.ds.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=MPent76.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=MJav77.ManJav2.0.ds.bam --output=/gscratch/wasser/kimh11/zoo-files/MJav77.ManJav2.0.ds.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=MJav77.ManJav2_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true

fgbio ClipBam --input=P442983.ManJav2.0.bam --output=P442983.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442983.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442984.ManJav2.0.bam --output=P442984.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442984.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442985.ManJav2.0.bam --output=P442985.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442985.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442986.ManJav2.0.bam --output=P442986.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442986.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442987.ManJav2.0.bam --output=P442987.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442987.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442988.ManJav2.0.bam --output=P442988.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442988.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442989.ManJav2.0.bam --output=P442989.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442989.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442990.ManJav2.0.bam --output=P442990.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442990.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442991.ManJav2.0.bam --output=P442991.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442991.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442992.ManJav2.0.bam --output=P442992.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442992.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442993.ManJav2.0.bam --output=P442993.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442993.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442994.ManJav2.0.bam --output=P442994.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442994.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442995.ManJav2.0.bam --output=P442995.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442995.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442996.ManJav2.0.bam --output=P442996.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442996.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442997.ManJav2.0.bam --output=P442997.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442997.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442998.ManJav2.0.bam --output=P442998.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442998.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442999.ManJav2.0.bam --output=P442999.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442999.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P443000.ManJav2.0.bam --output=P443000.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P443000.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P443001.ManJav2.0.bam --output=P443001.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P443001.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P336742.ManJav2.0.bam --output=P336742.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P336742.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P336743.ManJav2.0.bam --output=P336743.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P336743.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P336744.ManJav2.0.bam --output=P336744.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P336744.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P336745.ManJav2.0.bam --output=P336745.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P336745.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P336746.ManJav2.0.bam --output=P336746.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P336746.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P336747.ManJav2.0.bam --output=P336747.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P336747.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442962.ManJav2.0.bam --output=P442962.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442962.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442963.ManJav2.0.bam --output=P442963.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442963.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442964.ManJav2.0.bam --output=P442964.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442964.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442965.ManJav2.0.bam --output=P442965.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442965.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442966.ManJav2.0.bam --output=P442966.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442966.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442967.ManJav2.0.bam --output=P442967.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442967.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442968.ManJav2.0.bam --output=P442968.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442968.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442969.ManJav2.0.bam --output=P442969.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442969.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442970.ManJav2.0.bam --output=P442970.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442970.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442971.ManJav2.0.bam --output=P442971.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442971.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442972.ManJav2.0.bam --output=P442972.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442972.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442973.ManJav2.0.bam --output=P442973.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442973.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442974.ManJav2.0.bam --output=P442974.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442974.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442975.ManJav2.0.bam --output=P442975.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442975.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442976.ManJav2.0.bam --output=P442976.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442976.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442977.ManJav2.0.bam --output=P442977.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442977.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442978.ManJav2.0.bam --output=P442978.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442978.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442979.ManJav2.0.bam --output=P442979.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442979.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442980.ManJav2.0.bam --output=P442980.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442980.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442981.ManJav2.0.bam --output=P442981.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442981.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442982.ManJav2.0.bam --output=P442982.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442982.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442983.ManJav2.0.bam --output=P442983.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442983.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442984.ManJav2.0.bam --output=P442984.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442984.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442985.ManJav2.0.bam --output=P442985.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442985.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442986.ManJav2.0.bam --output=P442986.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442986.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442987.ManJav2.0.bam --output=P442987.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442987.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442988.ManJav2.0.bam --output=P442988.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442988.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442989.ManJav2.0.bam --output=P442989.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442989.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442990.ManJav2.0.bam --output=P442990.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442990.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442991.ManJav2.0.bam --output=P442991.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442991.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442992.ManJav2.0.bam --output=P442992.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442992.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442993.ManJav2.0.bam --output=P442993.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442993.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442994.ManJav2.0.bam --output=P442994.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442994.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442995.ManJav2.0.bam --output=P442995.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442995.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442996.ManJav2.0.bam --output=P442996.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442996.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442997.ManJav2.0.bam --output=P442997.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442997.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442998.ManJav2.0.bam --output=P442998.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442998.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P442999.ManJav2.0.bam --output=P442999.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P442999.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P443000.ManJav2.0.bam --output=P443000.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P443000.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
fgbio ClipBam --input=P443001.ManJav2.0.bam --output=P443001.ManJav2.0.clipped.bam --ref=/gscratch/wasser/kimh11/reference-genomes/ManJav2.0/GCF_014570535.1_YNU_ManJav_2.0_genomic.fasta --metrics=P443001.ManJav2.0_metrics.txt --clipping-mode=Soft --clip-overlapping-reads=true
